python/mlc_chat/compiler_pass/fuse_dequantize_matmul_epilogue.py

```diff
@@ -0,0 +1,322 @@
+"""A compiler pass that fuses dequantize matmul + epilogue."""
+import operator
+from functools import reduce
+
+import tvm
+from tvm import IRModule, relax
+from tvm.relax.dpl import rewrite_call
+from tvm.relax.dpl.pattern import is_op, wildcard
+
+
+@tvm.transform.module_pass(opt_level=0, name="FuseDequantizeEpilogue")
+class FuseDequantizeEpilogue:  # pylint: disable=too-few-public-methods
```

Comment:
How about we being more specific saying it's only applied to FT kernels? How about `FuseFTDequantizeEpilogue`?